accessibility-first-step/en/12 - When you provide a title attribute for an iframe element, do not leave it empty.md

```diff
@@ -3,20 +3,20 @@ id:         12
 priority:   Error
 topic:      HTML
 selector:   iframe[title]
-status:     not started
+status:     needs review
 ---
 
 # When you provide a title attribute for an iframe element, do not leave it empty
 
 ## Purpose
 
-Permettre la restitution d'un contenu décrivant la fonction et/ou le contenu de l'élément.
+Allow the restitution of a content describing the function and/or content of the element.

```

Comment:
Dans les PR #21 et #22 on a "the function and/or **the** content".
Pour harmoniser ajouter le `the` ici aussi.